go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/go-openapi/spec v0.19.3
 	github.com/gobuffalo/packr v1.30.1
 	github.com/gogo/protobuf v1.3.1
-	github.com/golang/mock v1.4.3
+	github.com/golang/mock v1.4.4-0.20200731163441-8734ec565a4d
```

Comment:
By intent?

Reply:
Yes, it is done to include golang/mock#458, which is currently only on master and not yet released.
(see the commit message of the first commit 164b9c5a0f6ead02118148683902f1b1fd4c74ec)

Reply:
I see, thanks!